### ISSUE TITLE
use googles API push data 

### ISSUE THREAD

**Issue by merryt** [merryt/Drive_to_JAM#11]
We currently pull all the data from google API, I heard that googles API can do PUSH

**Comment by cyface** [comment]
Yeah so we go all Drive API (and not Docs API). Docs here: https://developers.google.com/drive/api/v3/push
